docs/overview.adoc

```diff
@@ -265,6 +265,17 @@ GraphML is used by other tools, like Gephi and CytoScape to read graph data.
 | apoc.export.graphml.graph(graph,file,config) | exports given graph object as graphml to the provided file
 | apoc.export.graphml.query(query,file,config) | exports nodes and relationships from the Cypher statement as graphml to the provided file
 |===
+
+All `Point` or `Temporal` data types are exported formatted as a String
+
+e.g:
+[cols="1m,2"]
+|===
+|Point 3d | {"crs":"wgs-84-3d","latitude":56.7,"longitude":12.78,"height":100.0}
```

Comment:
good question, can gephi / other graphml tools work with that? 
can we quickly check if there is any precedent about geo-coordinates in graphml?

Reply:
We checked if there was a spatial support in new versions of GraphML, but it is not. So we decided to render it as a json String. We tested it with *Gephi* and *yEd* and is managed as simple string column.